### PR TITLE
chore: updating pt/es/fr translations

### PR DESCRIPTION
**Pull request by thesmithdao**
Updating shapeshift/lib with PT/ES/FR translations

~~ thesmith - Globalization Workstream

### CODE REVIEW

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
plz proofread - lots of spaces between `[] ()` in markdown, and spaces in URLs for `es.json` at least, haven't read others

**Review by thesmithdao — COMMENTED** [review]
fixing markdown

File: packages/asset-service/src/service/descriptions/es.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
... lo que les brinda el poder de crear aplicaciones únicas e innovadoras.<br/><br/>Ethereum es el primer [smart plataforma de contrato] (https://www.coingecko.com/en/categories/smart-contract-platform), han ganado mucha popularidad, lo que ha dado lugar a que nuevos competidores luchen por la cuota de mercado. Los competidores incluyen: [Ethereum Classic](https://www.coingecko.com/en/coins/ethereum-classic) que es la antigua cadena de Ethereum, [Qtum](https://www.coingecko.com/en/ monedas/qtum), [EOS](https://www.coingecko.com/en/coins/eos), [Neo](https://www.coingecko.com/en/coins/neo), [Icono](https://www.coingecko.com/en/coins/icon), [Tron](https://www.coingecko.com/en/coins/tron) y [Cardano](https://www.coingecko .com/en/coins/cardano).<br/><br/>Las billeteras Ethereum son bastante simples de configurar con múltiples opciones populares como [KeepKey](https://www.coingecko.com/learn/keepkey- bitcoin-hardware-wallet-review) y [MetaMask](https://www.coingecko.com/buzz/complete-beginners-guide-to-metamask?locale=en). Lea aquí para obtener más guías sobre el uso de la billetera Ethereum: [Cómo usar una billetera Ethereum](https://www.coingecko.com/buzz/how-to-use-an-ethereum-wallet)",
```
Comment (gomesalexandre) [inline packages/asset-service/src/service/descriptions/es.json:2]: - `[Qtum](https://www.coingecko.com/en/ monedas/qtum)` that looks no good to me. Additional space, still an invalid link if you remove it 
- `[Icono](https://www.coingecko.com/en/coins/icon)` checked the link on Coingecko, name doesn't match? Should be `ICON`
- `[KeepKey](https://www.coingecko.com/learn/keepkey- bitcoin-hardware-wallet-review)` there's a whitespace that shouldn't be there, and even if you remove it that's a dead link

File: packages/asset-service/src/service/descriptions/es.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
...9694714e2874c1ce77aa1e599461e18": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el grupo y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd). TerraUSD([UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd)) es la moneda estable algorítmica y descentralizada de la cadena de bloques Terra. Para adquirir Curve MIM-UST, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) en [Curve](https://curve.fi/factory/48/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanente, pero debido a que todos los activos en el fondo están vinculados a $1, la pérdida impermanente es poco probable y solo se produciría si uno de los activos en el pool perdió su anclaje.",
```
Comment (gomesalexandre) [inline packages/asset-service/src/service/descriptions/es.json:5]: UST is now USTC, pls update references accordingly 
image

File: packages/asset-service/src/service/descriptions/pt.json [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
+  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "Este token representa uma pool de liquidez na Curve, a corretora descentralizada líder no setor. Holders podem ganhar taxas dos usuários que negociam na pool e também podem depositar seus tokens LP na[ ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e [ETH](/assets/eip155:1/slip44:60). Para ter acesso ao LP da Curve de CRV-ETH, deposite [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e/ou [ETH](/assets/eip155:1/slip44:60) na [Curve](https://curve.fi/crveth/deposit). Por favor, saiba que as pools são compostas por ativos de preços de diferentes e estão sujeitas a perda impermanente. ",
```
Comment (gomesalexandre) [inline packages/asset-service/src/service/descriptions/pt.json:3]: Trailing whitespace, remove 
```suggestion
  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "Este token representa uma pool de liquidez na Curve, a corretora descentralizada líder no setor. Holders podem ganhar taxas dos usuários que negociam na pool e também podem depositar seus tokens LP na[ ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e [ETH](/assets/eip155:1/slip44:60). Para ter acesso ao LP da Curve de CRV-ETH, deposite [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e/ou [ETH](/assets/eip155:1/slip44:60) na [Curve](https://curve.fi/crveth/deposit). Por favor, saiba que as pools são compostas por ativos de preços de diferentes e estão sujeitas a perda impermanente.",
```

File: packages/asset-service/src/service/descriptions/pt.json [hunk 4]
```diff
@@ -1,9 +1,10 @@
...
+  "eip155:1/erc20:0x55a8a39bc9694714e2874c1ce77aa1e599461e18": "Este token representa uma pool de liquidez na Curve, uma corretora descentralizada líder de mercado. Os holders ganham taxas de usuários que negociam no pool e também podem depositar seus tokens LP na [ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) e [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd). TerraUSD([UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd)) é uma stablecoin algorítmica e descentralizada da blockchain Terra. Para adquirir o LP da Curve de MIM-UST, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) e [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) na [Curve](https://curve.fi/factory/48/deposit). Saiba que, como as pools de liquidez são compostas de ativos com preços diferentes, eles estão sujeitos a perdas temporárias, mas como todos os ativos do pool estão atrelados a $1, só ocorreria caso um dos ativos da pool perdesse a paridade.",
```
Comment (gomesalexandre) [inline packages/asset-service/src/service/descriptions/pt.json:5]: Ditto UST links linking to USTC

**Review by gomesalexandre — CHANGES_REQUESTED** [review]

File: packages/asset-service/src/service/descriptions/es.json (hunk 1 above)
Comment (thesmithdao) [reply packages/asset-service/src/service/descriptions/es.json:2]: Fixed, thanks.

File: packages/asset-service/src/service/descriptions/es.json (hunk 2 above)
Comment (thesmithdao) [reply packages/asset-service/src/service/descriptions/es.json:5]: Updating... en.json must be updated too

File: packages/asset-service/src/service/descriptions/pt.json (hunk 4 above)
Comment (thesmithdao) [reply packages/asset-service/src/service/descriptions/pt.json:5]: fixed UST -> USTC

File: packages/asset-service/src/service/descriptions/es.json (hunk 1 above)
Comment (gomesalexandre) [reply packages/asset-service/src/service/descriptions/es.json]: QTUM still links to a 404

File: packages/asset-service/src/service/descriptions/es.json [hunk 5]
```diff
@@ -1,7 +1,8 @@
...
...o inteligente, lo que les brinda el poder de crear aplicaciones únicas e innovadoras.<br/><br/>Ethereum es el primer [smart plataforma de contrato] (https://www.coingecko.com/en/categories/smart-contract-platform), han ganado mucha popularidad, lo que ha dado lugar a que nuevos competidores luchen por la cuota de mercado. Los competidores incluyen: [Ethereum Classic](https://www.coingecko.com/en/coins/ethereum-classic) que es la antigua cadena de Ethereum, [Qtum](https://www.coingecko.com/en/monedas/qtum), [EOS](https://www.coingecko.com/en/coins/eos), [Neo](https://www.coingecko.com/en/coins/neo), [ICON](https://www.coingecko.com/en/coins/icon), [Tron](https://www.coingecko.com/en/coins/tron) y [Cardano](https://www.coingecko.com/en/coins/cardano).<br/><br/>Las billeteras Ethereum son bastante simples de configurar con múltiples opciones populares como [KeepKey](https://www.coingecko.com/learn/keepkey-bitcoin-hardware-wallet-review) y [MetaMask](https://www.coingecko.com/buzz/complete-beginners-guide-to-metamask). Lea aquí para obtener más guías sobre el uso de la billetera Ethereum: [Cómo usar una billetera Ethereum](https://www.coingecko.com/buzz/how-to-use-an-ethereum-wallet)",
```
Comment (gomesalexandre) [inline packages/asset-service/src/service/descriptions/es.json:2]: Whitespace still present in `[smart plataforma de contrato] (https://www.coingecko.com/en/categories/smart-contract-platform)`

```suggestion
  "eip155:1/slip44:60": "Ethereum es una [plataforma de contratos inteligentes](https://www.coingecko.com/en/categories/smart-contract-platform) que permite a los desarrolladores crear tokens y aplicaciones descentralizadas (dapps). ETH es la moneda nativa de la plataforma Ethereum y también funciona como tarifa de transacción para los mineros en la red Ethereum.<br/><br/>Ethereum es el pionero en contratos inteligentes basados en blockchain. El contrato inteligente es esencialmente un código de computadora que se ejecuta exactamente según lo programado sin posibilidad de tiempo de inactividad, censura, fraude o interferencia de terceros. Puede facilitar el intercambio de dinero, contenido, propiedades, acciones o cualquier cosa de valor. Cuando se ejecuta en la cadena de bloques, un contrato inteligente se convierte en un programa informático autónomo que se ejecuta automáticamente cuando se cumplen condiciones específicas.<br/><br/>Ethereum permite a los programadores ejecutar contratos inteligentes completos que pueden personalizarse. En lugar de brindar un conjunto de operaciones limitadas, Ethereum les permite a los desarrolladores tener un control completo sobre la personalización de su contrato inteligente, lo que les brinda el poder de crear aplicaciones únicas e innovadoras.<br/><br/>Ethereum es el primer [smart plataforma de contrato](https://www.coingecko.com/en/categories/smart-contract-platform), han ganado mucha popularidad, lo que ha dado lugar a que nuevos competidores luchen por la cuota de mercado. Los competidores incluyen: [Ethereum Classic](https://www.coingecko.com/en/coins/ethereum-classic) que es la antigua cadena de Ethereum, [Qtum](https://www.coingecko.com/en/monedas/qtum), [EOS](https://www.coingecko.com/en/coins/eos), [Neo](https://www.coingecko.com/en/coins/neo), [ICON](https://www.coingecko.com/en/coins/icon), [Tron](https://www.coingecko.com/en/coins/tron) y [Cardano](https://www.coingecko.com/en/coins/cardano).<br/><br/>Las billeteras Ethereum son bastante simples de configurar con múltiples opciones populares como [KeepKey](https://www.coingecko.com/learn/keepkey-bitcoin-hardware-wallet-review) y [MetaMask](https://www.coingecko.com/buzz/complete-beginners-guide-to-metamask). Lea aquí para obtener más guías sobre el uso de la billetera Ethereum: [Cómo usar una billetera Ethereum](https://www.coingecko.com/buzz/how-to-use-an-ethereum-wallet)",
```

**Review by gomesalexandre — CHANGES_REQUESTED** [review]

File: packages/asset-service/src/service/descriptions/es.json [hunk 6]
```diff
@@ -1,7 +1,8 @@
...
...14e2874c1ce77aa1e599461e18": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el grupo y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [USTC](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd). TerraUSD([USTCC](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd)) es la moneda estable algorítmica y descentralizada de la cadena de bloques Terra. Para adquirir Curve MIM-USTC, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [USTC](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) en [Curve](https://curve.fi/factory/48/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanente, pero debido a que todos los activos en el fondo están vinculados a $1, la pérdida impermanente es poco probable y solo se produciría si uno de los activos en el pool perdió su anclaje.",
```
Comment (0xdef1cafe) [inline packages/asset-service/src/service/descriptions/es.json:5]:
```suggestion
  "eip155:1/erc20:0x55a8a39bc9694714e2874c1ce77aa1e599461e18": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el grupo y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [USTC](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd). TerraUSD([USTC](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd)) es la moneda estable algorítmica y descentralizada de la cadena de bloques Terra. Para adquirir Curve MIM-USTC, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [USTC](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) en [Curve](https://curve.fi/factory/48/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanente, pero debido a que todos los activos en el fondo están vinculados a $1, la pérdida impermanente es poco probable y solo se produciría si uno de los activos en el pool perdió su anclaje.",
```

File: packages/asset-service/src/service/descriptions/pt.json [hunk 7]
```diff
@@ -1,9 +1,10 @@
...
+  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "Este token representa uma pool de liquidez na Curve, a corretora descentralizada líder no setor. Holders podem ganhar taxas dos usuários que negociam na pool e também podem depositar seus tokens LP na[ ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e [ETH](/assets/eip155:1/slip44:60). Para ter acesso ao LP da Curve de CRV-ETH, deposite [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e/ou [ETH](/assets/eip155:1/slip44:60) na [Curve](https://curve.fi/crveth/deposit). Por favor, saiba que as pools são compostas por ativos de preços de diferentes e estão sujeitas a perda impermanente.",
```
Comment (0xdef1cafe) [inline packages/asset-service/src/service/descriptions/pt.json:3]:
```suggestion
  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "Este token representa uma pool de liquidez na Curve, a corretora descentralizada líder no setor. Holders podem ganhar taxas dos usuários que negociam na pool e também podem depositar seus tokens LP na [ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e [ETH](/assets/eip155:1/slip44:60). Para ter acesso ao LP da Curve de CRV-ETH, deposite [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e/ou [ETH](/assets/eip155:1/slip44:60) na [Curve](https://curve.fi/crveth/deposit). Por favor, saiba que as pools são compostas por ativos de preços de diferentes e estão sujeitas a perda impermanente.",
```

File: packages/asset-service/src/service/descriptions/es.json (hunk 1 above)
Comment (thesmithdao) [reply packages/asset-service/src/service/descriptions/es.json]: Changed the locale, from es to en. link is up

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

**Comment by thesmithdao** [comment]
Engineering team, I appreciate the support.
This is one of our first PRs and we will refine and polish our procedures to perfection.

**Review by thesmithdao — COMMENTED** [review]
submitting changes

**Review by thesmithdao — COMMENTED** [review]
submitting changes

File: packages/asset-service/src/service/descriptions/pt.json [hunk 8]
```diff
@@ -1,9 +1,10 @@
...
...a [ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) e [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa412cbac8115ee7e52bde6ee412cbac8115ee7e52bde6e). [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490) é outro LP da Curve, formado por [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/ erc20:0xdac17f958d2ee523a2206206994597c13d831ec7) e [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). Para adquirir crvMIM, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3), [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7) e/ou [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) na [Curve](https://curve.fi/mim/deposit). Saiba que, como as pools de liquidez são compostas de ativos com preços diferentes, eles estão sujeitos a perdas temporárias, mas como todos os ativos do pool estão atrelados a $1, só ocorreria caso um dos ativos da pool perdesse a paridade.",
```
Comment (0xdef1cafe) [inline packages/asset-service/src/service/descriptions/pt.json:4]:
```suggestion
  "eip155:1/erc20:0x5a6a4d54456819380173272a5e8e9b9904bdf41b": "Este token representa uma pool de liquidez na Curve, uma corretora descentralizada líder de mercado. Os holders ganham taxas de usuários que negociam no pool e também podem depositar seus tokens LP na [ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) e [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa412cbac8115ee7e52bde6ee412cbac8115ee7e52bde6e). [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490) é outro LP da Curve, formado por [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7) e [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). Para adquirir crvMIM, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3), [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7) e/ou [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) na [Curve](https://curve.fi/mim/deposit). Saiba que, como as pools de liquidez são compostas de ativos com preços diferentes, eles estão sujeitos a perdas temporárias, mas como todos os ativos do pool estão atrelados a $1, só ocorreria caso um dos ativos da pool perdesse a paridade.",
```

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

**Review by thesmithdao — COMMENTED** [review]
we identified many gaps in several parts, this raises a redflag on the gitlocalize formatting
thank you

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**